scripts/zones/Temple_of_Uggalepih/npcs/qm5.lua

```diff
@@ -11,7 +11,7 @@ local entity = {}
 entity.onTrade = function(player, npc, trade)
     if
         npcUtil.tradeHas(trade, xi.items.OFFERING_TO_UGGALEPIH) and
-        npcUtil.popFromQM(player, npc, { ID.mob.CRIMSON_TOOTHED_PAWBERRY, ID.mob.CRIMSON_TOOTHED_PAWBERRY + 2 })
+        npcUtil.popFromQM(player, npc, { ID.mob.CRIMSON_TOOTHED_PAWBERRY, ID.mob.CRIMSON_TOOTHED_PAWBERRY + 2 }, { claim = true })
```

Comment:
is the QM supposed to disappear as well?

Reply:
Honestly unsure of this. I was only digging into the claim issue, so I'm not certain.

Reply:
As it looks like it will not hide - need to add in the following:
{ claim = true, hide = 0 } with hide being seconds

see https://github.com/hookstar/AirSkyBoat/blob/staging/scripts/globals/npc_util.lua#L131

Reply:
This is fixed, just waiting on checks to run again.